app/src/main/res/layout/list_item_account_manager_placeholder.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are there any differences between this layout file and `list_item_app_shortcut_placeholder` other than the text? If not you could rename it into a more generic name and reuse it for both cases with text binding from code.